components/src/io_expander/stmpe1600/stmpe1600.adb

```diff
@@ -0,0 +1,338 @@
+------------------------------------------------------------------------------
+--                                                                          --
+--                    Copyright (C) 2017, AdaCore                           --
+--                                                                          --
+--  Redistribution and use in source and binary forms, with or without      --
+--  modification, are permitted provided that the following conditions are  --
+--  met:                                                                    --
+--     1. Redistributions of source code must retain the above copyright    --
+--        notice, this list of conditions and the following disclaimer.     --
+--     2. Redistributions in binary form must reproduce the above copyright --
+--        notice, this list of conditions and the following disclaimer in   --
+--        the documentation and/or other materials provided with the        --
+--        distribution.                                                     --
+--     3. Neither the name of STMicroelectronics nor the names of its       --
```

Comment:
If this is not derived from an STM file, please use the header like in https://github.com/AdaCore/Ada_Drivers_Library/blob/master/components/src/io_expander/MCP23xxx/mcp23x08.ads.